scripts/github/repositories.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2022 Greenbone Networks GmbH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from argparse import ArgumentParser, Namespace
+
+from pontos.github.api import GitHubAsyncRESTApi
+from pontos.github.models.organization import Repository
+
+
+def add_script_arguments(parser: ArgumentParser) -> None:
+    parser.add_argument("organization")
```

Comment:
```suggestion
    parser.add_argument("organization")
    
    parser.add_argument(
        "-t",
        "--type",
        help="what type of repositories should be printed? Default: %(default),
        default=RepositoryType.ALL
```

Maybe we can also add this arg, for e.g. only print public/private?

Reply:
Yes of course. This scripts are just a draft and starting point.